Sources/SwiftProtobuf/TextFormatEncodingVisitor.swift

```diff
@@ -96,13 +98,17 @@ internal struct TextFormatEncodingVisitor: Visitor {
               var bytes = Internal.emptyData
               try decoder.decodeSingularBytesField(value: &bytes)
               bytes.withUnsafeBytes { (p: UnsafePointer<UInt8>) -> () in
-                  var testDecoder = BinaryDecoder(forReadingFrom: p, count: bytes.count)
+                  var testDecoder = BinaryDecoder(forReadingFrom: p,
+                                                  count: bytes.count,
+                                                  parent: decoder)
```

Comment:
Since TextFormat should never be a _wire format_, I wasn't planning on added it.

My thought was to keep independent ones, just like the Error object, so they don't have to stay in sync and we don't have to try to document what applies where.  Also, if someone did want to write a custom decoder, they might need their own options so a custom options object wouldn't help them.